src/Commands/CommandClearInventory.cs

```diff
@@ -77,7 +77,10 @@ public class CommandClearInventory : EssCommand {
                 (byte) 1, (byte) 0, EMPTY_BYTE_ARRAY);
 
             // Remove items
-            for (byte page = 0; page < 8; page++) {
+            for (byte page = 0; page < PlayerInventory.PAGES; page++) {
+                if(page == PlayerInventory.AREA)
```

Comment:
What is this for?

Reply:
Well using magic constants like "8" is not a good idea.
If you mean the AREA page, its the near items page, and technically not part of the inventory

Reply:
Yes, I was talking about the AREA page.

Thanks.